CHANGELOG.md

```diff
@@ -1,159 +1,208 @@
-# Change Log
+# Changelog
 
-## Unreleased
+flake8-pyi uses Calendar Versioning (CalVer).
 
-* **Breaking change:** Previously, flake8-pyi monkey patched flake8's F821
-  (undefined name) check to avoid false positives in stub files. This monkey
-  patch has been removed. Instead, we now recommend to disable F821 when running flake8
-  on stub files.
+## [Unreleased]
+
+### New Error Codes
+
+* Y068 – Don't use `@override` in stub files
+
+### Other Changes
+
+* **Breaking change:** Previously, flake8-pyi monkey patched flake8's F821 (undefined name) check to
+  avoid false positives in stub files. This monkey patch has been removed.
+  Instead, we now recommend to disable F821 when running flake8 on stub files.
 * Remove the now unnecessary `--no-pyi-aware-file-checker` option.
```

Comment:
This probably also counts as a breaking change, actually. Maybe we should have a standardised `### Breaking changes` section for changes like this?